gravitee-apim-console-webui/src/services-ngx/api-v2.service.spec.ts

```diff
@@ -195,6 +195,24 @@ describe('ApiV2Service', () => {
     });
   });
 
+  describe('import', () => {
+    it('should call the API', (done) => {
+      const fakeApi = fakeApiV4();
+
+      apiV2Service.export(fakeApi.id).subscribe((api) => {
```

Comment:
This is calling "export" instead of "import" 🙃

Reply:
haha yes just see ci. fixed